frontend/src/index.css

```
@@ -326,7 +363,10 @@ code {
 
 .user-heading-row h3 {
   margin: 0;
-  font-family: Sora, Segoe UI, sans-serif;
+  font-family:
+    Sora,
+    Segoe UI,
```

Stylelint (17.5.0)

[error] Expected quotes around "Segoe UI" (font-family-name-quotes)

(font-family-name-quotes)